scala/lambdas/preingest-tdr-aggregator/README.md

```diff
@@ -0,0 +1,56 @@
+# DR2 Preingest TDR aggregator
+
```

Comment:
Do you think we should lift some/all of the information on the Lambda from [this paragraph](https://github.com/kurtismash/aws-serverless-unique-event-aggregator#:~:text=non%2Didempotent%20situations.-,The,-aggregation%20uses%20Lambda%27s) and add it here?

Reply:
Good point. I've replaced the original blurb with that paragraph.

Reply:
Yep, definitely worth it. Also, as I added it in, I noticed it talked about a condition expression on the assetId. I hadn't added that and I was calling it messageId instead of assetId. So I've changed messageId to assetId and added a conditional expression to the write call.